defect-tracker-server/src/main/java/com/defect/tracker/data/entities/Employee.java

```diff
@@ -0,0 +1,90 @@
+package com.defect.tracker.data.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Entity
+@Table(name = "employee")
+public class Employee {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	private String name;
+	private String designation;
```

Comment:
there is a table designation. you can it after merge designation to development.so then use designation-id

Reply:
ok